e2e_test/source/tvf/postgres_query.slt

```diff
@@ -0,0 +1,40 @@
+system ok
+PGHOST=db PGPORT=5432 PGUSER=postgres PGPASSWORD=postgres PGDATABASE=cdc_test psql -c "
```

Comment:
It would be better to use $RISEDEV_ envvars, instead of hard-coded ones. Then you can run it easier locally

Reply:
Current e2e-source-test does not have a postgres service managed by risedev. We can refactor it after https://github.com/risingwavelabs/risingwave/pull/18099.

Reply:
I think we already have risedev managed pg https://github.com/risingwavelabs/risingwave/issues/16662

Reply:
Yes, it can be configured as a service in the risedev config. But switching the risedev configuration may cause diff conflicts with #18099. There could be conflicts with the ci-managed postgres instance as well.

Migrating everything together in #18858 after #18099 will be more straightforward.

Reply:
Hmmm, we already have a dedicated risedev profile, so I don't think it will affect other stuff.. 

https://github.com/risingwavelabs/risingwave/blob/c80cbf34003a9b9f3aaf77beae3cf8958d55bc6d/ci/scripts/e2e-source-test.sh#L38-L41

https://github.com/risingwavelabs/risingwave/blob/c80cbf34003a9b9f3aaf77beae3cf8958d55bc6d/risedev.yml#L952-L979

Reply:
I just mean to write the new tests introduced in the new style, without changing existing tests. If you mean migrating all tests to the style later after that PR, it also sounds good